### PR TITLE
Implement contracts overview page table

### PR DESCRIPTION
**Pull request by MartinForre**

### CODE REVIEW

File: src/frontend/src/pages/ProjectPage/pages/Columns.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        label: 'Name',
+        sortable: true,
+        component: ({ item }) => (
+            <ContractLinkColumn contractId={item.id}>{item.name}</ContractLinkColumn>
+        ),
+    },
+    {
+        accessor: 'companyRepPositionId',
+        key: 'companyRepPositionId',
+        label: 'Eqinor company rep',
```
Comment (olerichard) [inline src/frontend/src/pages/ProjectPage/pages/Columns.tsx]: spelling

File: src/frontend/src/pages/ProjectPage/pages/Columns.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        component: ({ item }) => <PositionColumn positionId={item.companyRepPositionId} />,
+    },
+    {
+        accessor: 'contractResponsiblePositionId',
+        key: 'contractResponsiblePositionId',
+        label: 'Eqinor contract rep',
```
Comment (olerichard) [inline src/frontend/src/pages/ProjectPage/pages/Columns.tsx]: spelling

File: src/frontend/src/pages/ProjectPage/pages/Columns.tsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
+import * as React from 'react';
```
Comment (eskil0312) [inline src/frontend/src/pages/ProjectPage/pages/Columns.tsx]: How about scoping Columns.tsx and ContrancOverviewPage.tsx to a sub-folder?

Comment (MartinForre) [reply src/frontend/src/pages/ProjectPage/pages/Columns.tsx]: Hmm. Could move the entire contents of the pages folder to "ContractsOverviewPage"

**Pull request merged**